tutorials/distributed_data_classification/distributed_data_classification.ipynb

```diff
@@ -52,8 +50,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "cluster = LocalCUDACluster(rmm_async=True, rmm_pool_size=\"1GB\")\n",
-    "client = Client(cluster)"
+    "client = get_client()"
```

Comment:
You probably want to set `get_client(cluster_type="gpu")`, right?

Reply:
Hmm when I add that I get:
```
Exception: NotImplementedError('\n        NeMo Curator does not support query planning yet.\n        Please disable query planning before importing\n        `dask.dataframe` or `dask_cudf`. This can be done via:\n        `export DASK_DATAFRAME__QUERY_PLANNING=False`, or\n        importing `dask.dataframe/dask_cudf` after importing\n        `nemo_curator`.\n        ')
```
?

Reply:
Okay, I updated to the latest NeMo Curator changes but am still seeing the same error for some reason. When I add `%env DASK_DATAFRAME__QUERY_PLANNING=False` to the first cell, I'm able to run almost the entire notebook but fails at the last cell with a
```
KilledWorker: Attempted to run task ('read_single_partition-98560c3028597b26370b5f8da46466b3', 0) on 4 different workers, but all those workers died while running it. The last worker that attempt to run the task was tcp://127.0.0.1:35335. Inspecting worker logs is often a good next step to diagnose what went wrong. For more information see https://distributed.dask.org/en/stable/killed.html.
```

Reply:
NVM, it looks like it was an issue with too many other tasks running at once. Updated!